### PR TITLE
Add Ituran integration

### PR DESCRIPTION
**Pull request by shmuelzon**
## Proposed change

This PR adds documentation to a new [Ituran](http://www.ituran.com) integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/129067
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/6010

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for the Ituran integration within Home Assistant.
	- Users can now retrieve vehicle information from Ituran-equipped vehicles.
	- Added guidance for setup requirements, including necessary user inputs for configuration.
	- Detailed explanation of the device tracker platform for tracking vehicle locations.
	- Included links for downloading the Ituran app on Android and iOS.
	- Noted limitations regarding the use of the official app during integration configuration.

### CODE REVIEW

**Comment by shmuelzon** [comment]
@coderabbitai review

File: source/_integrations/ituran.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: Ituran
+description: Instructions on how to add Ituran to Home Assistant.
+ha_category:
+  - Car
+  - Device Tracker
+  - Presence detection
+ha_release: '2025.01'
```
Comment (joostlek) [inline source/_integrations/ituran.markdown]:
```suggestion
ha_release: '2025.1'
```

File: source/_integrations/ituran.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+ID or passport number:
+  description: Your government ID or passport number used to sign-up with Ituran.
+Mobile phone number:
+  description: The mobile phone number used to sign-up with Ituran. A one-time-password will be sent to this number.
+Mobile ID:
+  description: A unique identifier used for registration with Ituran services. Leave this blank to generate a new ID automatically. Only provide a value if you want to share an existing Mobile ID from another device (e.g., your phone's Ituran app) to keep both devices active simultaneously.
```
Comment (joostlek) [inline source/_integrations/ituran.markdown]: Can be removed now

**Review by joostlek — DISMISSED** [review]
For the reviewer, please take the quality scale into account :)
https://github.com/home-assistant/core/pull/129067/files#diff-ffa985fe6bcfd9cdd30716e4b49f4b8f4cc21cbaaf93607e5dc9699961e86cd0

File: source/_integrations/ituran.markdown (hunk 2 above)
Comment (shmuelzon) [reply source/_integrations/ituran.markdown]: It's weird that you still see this as I already removed it 🤷‍♂️

File: source/_integrations/ituran.markdown (hunk 1 above)
Comment (shmuelzon) [reply source/_integrations/ituran.markdown]: Fixed

File: source/_integrations/ituran.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ha_platforms:
+  - device_tracker
+ha_integration_type: integration
+---
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
+
+{% note %}
+The information is pulled every 5 minutes from the Ituran web service; however, the data is updated at intervals determined by Ituran based on whether the vehicle is stationary or not.
+{% endnote %}
```
Comment (frenck) [inline source/_integrations/ituran.markdown]: We don't have to add a big note in the top for information like this.

Instead, display it as part of the documentation below, here is an example as documented in our developer documentation: <https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-data-update>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ituran.markdown (hunk 3 above)
Comment (shmuelzon) [reply source/_integrations/ituran.markdown]: Done. Also moved it to be below the prerequisites, seemed like a better flow 🤷‍♂️

File: source/_integrations/ituran.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
+
+## Prerequisites
+
+You must have an Ituran account for use with the Ituran APP ([Android](https://play.google.com/store/apps/details?id=com.ituran.forall)/[iOS](https://apps.apple.com/app/id1227756834)). All devices that work with the app, should work with this integration as well.
+
+## Data updates
+
+The information is pulled every 5 minutes from the Ituran web service; however, the data is updated at intervals determined by Ituran based on whether the vehicle is stationary or not.
```
Comment (frenck) [inline source/_integrations/ituran.markdown]: Let's move this section below the configuration.

File: source/_integrations/ituran.markdown [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+ID or passport number:
+  description: Your government ID or passport number used to sign-up with Ituran.
+Mobile phone number:
+  description: The mobile phone number used to sign-up with Ituran. A one-time-password will be sent to this number.
+{% endconfiguration_basic %}
+
+## Platforms
```
Comment (frenck) [inline source/_integrations/ituran.markdown]:
```suggestion
## Supported functionality
```

File: source/_integrations/ituran.markdown [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+### Device tracker
+
+The Ituran {% term integration %} will track the location of each vehicle registered to your account.
+
+## Known limitations
+
+- While this integration is configured with your account, you won't be able to use the official app, as only one connection at a time is supported
+
+{% include integrations/remove_device_service.md %}
```
Comment (frenck) [inline source/_integrations/ituran.markdown:52]: Let's use the standard as put here: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-removal-instructions

File: source/_integrations/ituran.markdown [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+ha_config_flow: true
+ha_codeowners:
+  - '@shmuelzon'
+ha_domain: ituran
+ha_platforms:
+  - device_tracker
+ha_integration_type: integration
+---
+
+The **Ituran** {% term integration %} allows you to retrieve information from your Ituran-equipped vehicle. It pulls information from the Ituran web service regarding the vehicle's location.
```
Comment (frenck) [inline source/_integrations/ituran.markdown]: This is expected to link to the upstream product or service, see also: 

https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-high-level-description

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ituran.markdown (hunk 6 above)
Comment (shmuelzon) [reply source/_integrations/ituran.markdown:52]: I took the standard but didn't include the last sentence:
> After deleting the integration, go to the app of the manufacturer and remove the Home Assistant integration from there as well.

As I don't think it's relevant here.

**Review by frenck — APPROVED** [review]
Thanks, @shmuelzon 👍 

../Frenck

**Pull request merged**